Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -381,8 +384,8 @@ def get_indicators_to_format(indicator_searcher: IndicatorsSearcher,
             # NG + XSIAM can recover from a shutdown
             if version.get('platform') == 'x2' or is_demisto_version_ge('8'):
                 raise SystemExit('Encountered issue in Elastic Search query. Restarting container and trying again.')
-
-    demisto.debug(f"Completed IOC search & format, found {ioc_counter} IOCs.")
+    if extensive_logging:
+        demisto.debug(f"edl: Completed IOC search & format, found {ioc_counter} IOCs. Their contents: {indicators}")
```

Comment:
in a separate log line, please print the length of indicators or ioc counter.
this should be printed regardless of extensive_logging

Reply:
added